src/frontend/src/scheduler/hummock_snapshot_manager.rs

```diff
@@ -49,11 +52,12 @@ enum EpochOperation {
 
 impl HummockSnapshotManager {
     pub fn new(meta_client: Arc<dyn FrontendMetaClient>) -> Self {
-        // do not use unbounded_channel because it may cause OOM when the RPC `get_epoch` blocks a
-        // long time.
         let (sender, mut receiver) = channel(MAX_WAIT_EPOCH_REQUEST_NUM);
+        let max_committed_epoch = Arc::new(AtomicU64::new(0));
```

Comment:
Use `AtomicU64::new(INVALID_EPOCH)` instead.

Reply:
```rust
impl HummockSnapshotManagerCore {
    fn new(meta_client: Arc<dyn FrontendMetaClient>) -> Self {
        Self {
            // Initialize by setting `is_outdated` to `true`.
            meta_client,
            epoch_to_query_ids: BTreeMap::default(),
            last_unpin_snapshot: Arc::new(AtomicU64::new(0)),
        }
    }

    ...
}
```

Should we change `last_unpin_snapshot` to `AtomicU64::new(INVALID_EPOCH)` as well?

Reply:
Also, the type of epoch is `HummockEpoch` on meta's side and `u64` on frontend's side. I wonder if that's intentional?

Reply:
I think we can start unify them in other PRs.

Reply:
> I think we can start unify them in other PRs.
I will do that.